tools/barista/src/utils/fetch-strapi-content.ts

```diff
@@ -86,8 +88,8 @@ async function fetchContent(requestPath: string): Promise<any> {
       });
 
       res.on('end', () => {
-        const jsonRes = JSON.parse(fullData);
-        if (jsonRes.length > 0) {
+        if (fullData.length > 0) {
```

Comment:
NIT: fullData.length would checks automatically if the length is greater than 0 ;)